2022-02-24 22:10:44,215 INFO    MainThread:25553 [wandb_setup.py:_flush():71] setting env: {}
2022-02-24 22:10:44,215 INFO    MainThread:25553 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-24 22:10:44,215 INFO    MainThread:25553 [wandb_init.py:_log_setup():371] Logging user logs to /home/simon/CDE_UBS/thesis/template/wandb/run-20220224_221044-227kcz9g/logs/debug.log
2022-02-24 22:10:44,215 INFO    MainThread:25553 [wandb_init.py:_log_setup():372] Logging internal logs to /home/simon/CDE_UBS/thesis/template/wandb/run-20220224_221044-227kcz9g/logs/debug-internal.log
2022-02-24 22:10:44,215 INFO    MainThread:25553 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7efc3f2e9fd0>
2022-02-24 22:10:44,216 INFO    MainThread:25553 [wandb_init.py:init():404] calling init triggers
2022-02-24 22:10:44,216 INFO    MainThread:25553 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-24 22:10:44,216 INFO    MainThread:25553 [wandb_init.py:init():460] starting backend
2022-02-24 22:10:44,217 INFO    MainThread:25553 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-24 22:10:44,287 INFO    MainThread:25553 [backend.py:ensure_launched():216] starting backend process...
2022-02-24 22:10:44,349 INFO    MainThread:25553 [backend.py:ensure_launched():222] started backend process with pid: 25598
2022-02-24 22:10:44,352 INFO    MainThread:25553 [wandb_init.py:init():469] backend started and connected
2022-02-24 22:10:44,402 INFO    MainThread:25553 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-24 22:10:44,403 INFO    MainThread:25553 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-24 22:10:44,403 INFO    MainThread:25553 [wandb_init.py:init():533] updated telemetry
2022-02-24 22:10:44,530 INFO    MainThread:25553 [wandb_init.py:init():563] communicating current version
2022-02-24 22:10:46,611 INFO    MainThread:25553 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-24 22:10:46,613 INFO    MainThread:25553 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-24 22:10:46,847 INFO    MainThread:25553 [wandb_init.py:init():606] starting run threads in backend
2022-02-24 22:10:51,866 INFO    MainThread:25553 [wandb_run.py:_console_start():1810] atexit reg
2022-02-24 22:10:51,868 INFO    MainThread:25553 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-24 22:10:51,868 INFO    MainThread:25553 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-24 22:10:51,872 INFO    MainThread:25553 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-24 22:10:51,872 INFO    MainThread:25553 [wandb_init.py:init():633] run started, returning control to user process
2022-02-24 22:10:59,238 INFO    MainThread:25553 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-24 22:10:59,238 INFO    MainThread:25553 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-24 22:13:50,621 INFO    MainThread:25553 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-24 22:13:50,622 INFO    MainThread:25553 [wandb_run.py:_restore():1752] restore
2022-02-24 22:13:51,572 INFO    MainThread:25553 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-24 22:13:51,782 INFO    MainThread:25553 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-24 22:13:51,947 INFO    MainThread:25553 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 27051
}

2022-02-24 22:13:52,052 INFO    MainThread:25553 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 27051
}

2022-02-24 22:13:52,154 INFO    MainThread:25553 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 27051
}

2022-02-24 22:13:52,257 INFO    MainThread:25553 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 27051
  total_bytes: 27051
}

2022-02-24 22:13:52,362 INFO    MainThread:25553 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 27051
  total_bytes: 27051
}

2022-02-24 22:13:52,465 INFO    MainThread:25553 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 27051
  total_bytes: 27051
}

2022-02-24 22:13:52,568 INFO    MainThread:25553 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 27051
  total_bytes: 27051
}

2022-02-24 22:13:52,671 INFO    MainThread:25553 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 27051
  total_bytes: 27051
}

2022-02-24 22:13:52,774 INFO    MainThread:25553 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 27051
  total_bytes: 27051
}

